modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc

```diff
@@ -9,7 +9,23 @@ Clusters using a restricted network must import the default must-gather image to
 
 .Procedure
 
-* Import the default must-gather image from your installation payload:
+. If you have not added your mirror registry's trusted CA to your cluster's image configuration object as part of the Samples Operator configuration, do so:
+.. Create the cluster’s image configuration object:
```

Comment:
I don't see this phrase elsewhere in the docs: _cluster's image configuration object_. 
I see _image configuration_. Should this be _image configuration custom resource (CR)_ or _image.config.openshift.io/cluster custom resource (CR)_, the latter [used here](https://docs.openshift.com/container-platform/4.7/openshift_images/image-configuration.html)?

Reply:
https://github.com/openshift/openshift-docs/blame/master/modules/installation-restricted-network-samples.adoc#L80